src/rpc/misc.cpp

```diff
@@ -66,6 +66,7 @@ UniValue getinfo(const JSONRPCRequest& request)
             "  \"paytxfee\": x.xxxx,         (numeric) the transaction fee set in " + CURRENCY_UNIT + "/kB\n"
             "  \"relayfee\": x.xxxx,         (numeric) minimum relay fee for transactions in " + CURRENCY_UNIT + "/kB\n"
             "  \"errors\": \"...\"           (string) any error messages\n"
+            "  \"deprecation-warning\": \"...\" (string) warning that the getinfo command is deprecated and will be removed in 0.16"
```

Comment:
Would slightly prefer if this was moved all the way up (pushed back as first string into the dict), such that it is harder to overlook.